src/menhir_parser.mly

```diff
@@ -40,10 +41,10 @@ let translate_old_rule : old_p_rule -> p_rule = fun r ->
     (if !verbose > 1 then Option.iter fn ao); x
   in
   let ctx = List.map get_var ctx in
-  (* Find the minimum number of arguments a variable is applied to. *)
   let is_pat_var env x =
     not (List.mem x env) && List.exists (fun y -> y.elt = x) ctx
   in
+  (* Find the maximum number of arguments a variable is applied to. *)
```

Comment:
This comment does not make more sense here as it did above. Again, please, avoid unnecessary changes.